src/contents/shared/ProfileEdit.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Component } from 'react';
+import fire from '../../config/Fire';
+import { ProfileEditView } from './ProfileEditView';
+import firebase from 'firebase';
+
+export class ProfileEdit extends Component {
+	state = {
+		  image: "https://example.com/jane-q-user/profile.jpg",
+		  url: this.props.user.photoURL,
+		  progress: 0,
+		  name: this.props.user.displayName,
+		  email: this.props.user.email,
+		  type: this.props.user.type,
+		  location: this.props.user.location,
+		  photoURL: this.props.user.url,
```

Comment:
why both url and photoURL?? aren't they supose to be the same . the avatar url, i took a look into the database we do not save things under the url, we save the photoURL

Reply:
Done! i have cleaned the code